src/Components/Map.jsx

```diff
@@ -2,6 +2,8 @@ import React, { Component } from 'react';
 import GoogleMapReact from 'google-map-react';
 import MapStyle from '../Modules/MapStyle'
 import { Icon } from 'semantic-ui-react';
+import Popup from 'reactjs-popup';
+import PostForm from './PostForm'
```

Comment:
This naming concerns me a bit. I would consider `CreateImageEntry` or something. `Post` indicates the HTTP action verb and Form is coupled to the HTML element.... Not sure about this....

Reply:
Makes sense.. Also since the component at this stage is not so much of a `form` anyway,..  Will think of something else. Possibly `ImageUpload`or similar.